schematics/scully/src/create-blog/index.ts

```diff
@@ -20,28 +20,13 @@ publish: false
 `);
         context.logger.info(`✅️Blog ${fullDay}-${name} file created`);
       }
-/*
-      // add into scully config
-      try {
-        const content: Buffer | null = host.read(`/scully.json`);
-        let jsonContent;
-        if (content) { jsonContent = JSON.parse(content.toString()); }
-        /* tslint:disable:no-string-literal
-        jsonContent.routes['/blog/:slug'] = {
-          type: 'contentFolder',
-            slug: {
-              folder: './blog'
-          }
-        };
-        host.overwrite(`/scully.json`, JSON.stringify(jsonContent, undefined, 2));
-        context.logger.info('✅️ Update scully.json');
-      } catch (e) {
-        context.logger.error('Cant update scully.json');
-      }
-*/
+
       let scullyJson;
       try {
         scullyJson = (host.read('/scully.config.js')).toString();
+        const newScullyJson = addRouteToScullyConfig(scullyJson, {name: 'blog', slug: 'slug', type: 'contentFolder'});
```

Comment:
This line of code looks wrong. 

Instead of `slug: 'slug'` shouldn't it be `slug: { folder: './blog' }`?

Reply:
This is ok, is a function, if you check  https://github.com/scullyio/scully/pull/15/files#diff-d0378fbbb13cbe2aa7db20719b215a5bL7 this format is in there